lib/index.ts

```diff
@@ -1,36 +1,36 @@
-import * as path from 'path';
-import * as fs from 'fs';
+import {parseConfig} from './parser';
+
+interface DepDict {
+  [dep: string]: DepTree;
+}
 
 export interface DepTree {
   name: string;
   version: string;
-  dependencies: {
-    [dep: string]: DepTree;
-  };
+  dependencies?: DepDict;
   depType?: 'prod' | 'dev';
   hasDevDependencies?: boolean;
   missingLockFileEntry?: boolean;
 }
 
+export type PackageManagerType = 'dep'|'govendor';
```

Comment:
let's adhere to the ones already in CLI s/dep/golangdep

Reply:
good idea.